ajax/libs/jodit/package.json

```
@@ -10,18 +10,36 @@
     "rich",
     "jodit"
   ],
-  "license": "CC-BY-NC-3.0",
+  "license": "MIT",
```

```suggestion
  "license": "GPL-2.0-or-later",
```

The main license in the source repository appears to be GNU GPL 2.0?

---

Main MIT @MattIPv4

---

Ah, cool.